src/main/java/net/huskycraft/blockyarena/BlockyArena.java

```diff
@@ -122,15 +148,48 @@ private void createDirectories() {
             }
         }
     }
+    
+    private void HandleConfiguration()
```

Comment:
Consider moving the following to a standalone class (a class dedicated for configuration loading). Right now the main class is overwhelmed.

Reply:
Done : 
-> ConfigManager Singleton Class
-> create config file if not exist and fill default node value
-> load the conf file if it exist !